controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook_test.go

```diff
@@ -639,7 +639,9 @@ func TestPaths(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			g := NewWithT(t)
-			g.Expect(paths(tt.path, tt.diff)).To(Equal(tt.expected))
+			for _, path := range paths(tt.path, tt.diff) {
+				g.Expect(path).To(BeElementOf(tt.expected))
```

Comment:
I'm sure this works but did you try [`ConsistOf`](https://onsi.github.io/gomega/#consistofelement-interface)?

Reply:
Done thanks!